libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -344,11 +344,17 @@ export class LuisRecognizer implements LuisRecognizerTelemetryClient {
     }
 
     // Gets a value indicating whether determines whether to log personal information that came from the user.
+    /**
+     *@returns True if will log personal information into the BotTelemetryClient.TrackEvent method; otherwise the properties will be filtered.
```

Comment:
```suggestion
     * @returns True if will log personal information into the BotTelemetryClient.TrackEvent method; otherwise the properties will be filtered.
```